man/logind.conf.xml

```diff
@@ -344,6 +344,15 @@
         are excluded from the effect of this setting. Defaults to <literal>yes</literal>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>StopIdleSessionSec=</varname></term>
+
+        <listitem><para>Specifies the periodic timeout after which <filename>systemd-logind</filename> checks idle state
+        of all sessions. Every sessions that is idle for longer then the timeout will be stopped. Defaults to
+        <literal>infinity</literal>, i.e. <filename>systemd-logind</filename> is not checking the idle state of sessions.
```

Comment:
We usually add a link to systemd.time man page for details on how to write times. WE usually also say that it takes an integer value which will be taken as seconds, or alternative any time specification that matches what systemd.time says with any time units of the user's choosing.